src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/OverseasEntitiesService.java

```diff
@@ -149,7 +150,10 @@ public ResponseEntity<Object> updateOverseasEntity(Transaction transaction,
         updateOverseasEntitySubmissionWithMetaData(overseasEntitySubmissionDao, submissionUri, requestId, userId);
 
         // Update company name set on the transaction, to ensure it matches the value received with this OE submission
-        transaction.setCompanyName(overseasEntitySubmissionDto.getEntityName());
+        if(Objects.nonNull(overseasEntitySubmissionDto.getEntityName())) {
```

Comment:
If we perform an update and don't supply a name (block or value), should we be updating the transaction with an empty string for name?

Reply:
Updated as above.